v2.2/delete.md

```diff
@@ -58,7 +57,7 @@ If disk usage is a concern, there are two potential solutions. The
 first is to [reduce the time-to-live](configure-replication-zones.html)
 (TTL) for the zone, which will cause garbage collection to clean up
 deleted rows more frequently. Second, unlike `DELETE`,
-[truncate](truncate.html) immediately deletes the entire table, so
```

Comment:
Huh, I think maybe GH ate my comment. Maybe @vivekmenezes can chime in here, but AFAIK, TRUNCATE doesn't delete the on-disk data any sooner -- it just avoids _writing_ deletes for every row.

Reply:
I think we want to says.

1. Normally, TRUNCATE should be used to delete all the table data, but TRUNCATE is implemented as a schema change which is not transactional. Therefore some data added within a short interval after the TRUNCATE might get deleted by the truncate.
2. Alternatively, DELETE can be use for transactional deletion of table data, but care must be taken because it will not work on very large tables because of the large transactions are not supported limitation.

Reply:
My point is that if you want the disk space back, _neither_ TRUNCATE nor DELETE are going to give it to you, but the current text seems to imply that TRUNCATE will which is what I was objecting to. To be fair, it is _better_ than `DELETE`s in terms of disk space since it doesn't write a tombstone on every row, but it still isn't going to free it up any sooner than the GC time.

Reply:
Thanks David and Vivek for the info.  Sounds like neither statement is useful for reclaiming disk, so we need to remove that bit.

Just pushed a change (also copied below) to reflect that neither statement will give your disk space back (soon), and you must update your GC TTL settings.  Let me know what you think.

> If disk usage is a concern, the solution is to [reduce the time-to-live](configure-replication-zones.html) (TTL) for the zone by setting `gc.ttlseconds` to a lower value, which will cause garbage collection to clean up deleted objects (rows, tables) more frequently.